.circleci/config.yml

```diff
@@ -0,0 +1,53 @@
+# We could use an image with pyenv & tox, but it's just as much work as doing a matrix build.
+# Sorry tox.ini...
+version: 2
+
+shared: &shared
+    steps:
+      - checkout
+      # Define config in CircleCi's project settings → Environment Variables
+      - run: echo \{\"app_id\":\"$FB_APP_ID\",\"app_secret\":\"$FB_APP_SECRET\",\"act_id\":\"act_$FB_ACCOUNT_ID\",\"page_id\":\"$FB_PAGE_ID\",\"access_token\":\"$FB_ACCESS_TOKEN\",\"business\":\"$FB_BUSINESS_ID\",\"sec_business\":\"$FB_BUSINESS_ID_BIS\",\"sec_act_id\":\"act_$FB_ACCOUNT_ID_BIS\"\} > config.json
+      # Cheapo virtualenv, CCi images use the circleci user so no writing to the root site-packages
+      - run: virtualenv venv
+      - run: venv/bin/python setup.py install
+      - run: venv/bin/python -m facebook_business.test.unit
+      - run: venv/bin/python -m facebook_business.test.integration
+
+jobs:
+  # Note: no 3.3 image from CCi readily available – not that we care about that version anyway.
```

Comment:
It's deprecated.

https://www.python.org/dev/peps/pep-0398/#x-end-of-life

> September 29, 2017

Reply:
As it should be! Upstream needs to update their `tox.ini`...